addons/hr_holidays/security/hr_holidays_security.xml

```diff
@@ -138,11 +138,14 @@
     <record id="hr_leave_allocation_rule_multicompany" model="ir.rule">
         <field name="name">Time Off: multi company global rule</field>
         <field name="model_id" ref="model_hr_leave_allocation"/>
+        <!-- holiday_status_id is required, but may not be set when creating a new record -->
```

Comment:
Can you elaborate on this? :thinking: 

AFAIK rule is evaluated on records, after you save the allocation form, for example. So, this is unnecessary, because holiday_status_id is required and it won't be left empty

Reply:
We found the issue in the code that was broken.